Not sure why the Azure pipelines tests are failing - @ericpre?
```
ImportError while loading conftest '/home/vsts/work/1/s/hyperspy/conftest.py'.
hyperspy/conftest.py:20: in <module>
    from traits.etsconfig.api import ETSConfig
E   ModuleNotFoundError: No module named 'traits'
```

--- a/setup.cfg
+++ b/setup.cfg
@@ -47,8 +47,10 @@ per-file-ignores =
     hyperspy/_lazy_signals.py:F401
     hyperspy/api.py:F401
     hyperspy/api_nogui.py:F401
+    hyperspy/conftest.py:F401
     hyperspy/datasets/*:F401
     hyperspy/drawing/widgets.py:F401
+    hyperspy/events.py:F401
     hyperspy/extensions.py:F401
     hyperspy/misc/machine_learning/import_sklearn.py:F401
     hyperspy/samfire_utils/*:F401

